* strsep is not available on Solaris, but mutt ships with replacement
  code.  Make the signature known though, as Gentoo on 64-bits platforms
  doesn't like warnings about "implicit declarations" of functions.

--- rfc3676.c
+++ rfc3676.c
@@ -37,6 +37,11 @@
 #include "ascii.h"
 #include "lib.h"
 
+#if !HAVE_STRSEP
+/* from included strsep.c */
+char *strsep (char **stringp, const char *delim);
+#endif
+
 #define FLOWED_MAX 77
 
 static int get_quote_level (const char *line)
